basic/boundCheck40.c

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016, 2017 National University of Singapore
+ * Copyright 2017 National University of Singapore
  *
  * This program is for testing memory bounds check interpolation: it
  * should have subsumptions not just at/near the end of the traces.
  * TRACER-X can analyze but LLBMC and KLEE can't analyze this program.
```

Comment:
@rasoolmaghareh The ending period has to be removed here as the sentence continues to the next line.